Makefile

```diff
@@ -55,7 +55,7 @@ installdocker:
 	docker-compose run --user=$(UID) --rm php composer install && chmod +x tests/yii
 
 testdocker:
-	docker-compose run --user=$(UID) --rm php sh -c 'vendor/bin/phpunit'
+	docker-compose run --user=$(UID) --rm php sh -c 'vendor/bin/phpunit --repeat 3'
```

Comment:
why repeat?

Reply:
Because of https://github.com/cebe/yii2-openapi/issues/139

To ensure no stale migration is present in DB.

A passing test if ran more than once, it must pass.